Week2/ramazanInce.js

```diff
@@ -0,0 +1,158 @@
+console.log(`
+Question 1`);
+// 1. Write a console.log statement saying
+// "Hello World!" for each language that you know.
+
+console.log(`Hello World
+Merhaba Dunya
+Priviyet Mir
+Marhaban Dunya
+Hallo Wereld
+`);
+
+console.log(`
+Question 2`);
+// 2. Consider the following code:
+
+console.log("I'm awesome");
```

Comment:
Just for your information, you can also use escape characters(using backslash) like this:
`console.log('I\'m awesome');`

The middle single quote will be then interpreted by JS as a string character and not a special character.

Reply:
Yes, I know that. I believe most of our class mate may use escape character then to have a different answer I did in that way.